diff --git a/keyboards/mechwild/puckbuddy/config.h b/keyboards/mechwild/puckbuddy/config.h
index 666c1ff085..a451afa8af 100644
--- a/keyboards/mechwild/puckbuddy/config.h
+++ b/keyboards/mechwild/puckbuddy/config.h
@@ -38,6 +38,10 @@
 /* set the tapping term for glidepoint pad to register a tap click */
 #define CIRQUE_PINNACLE_TAPPING_TERM 0 // This is set unreasonably low to effectively disable it
 
+/* set attenuation/speed to be more sensitive (kmn local tweak) */
+#define CIRQUE_PINNACLE_ATTENUATION ADC_ATTENUATE_2X
+#define GLIDEPOINT_DPI_DEFAULT 3
+
 /*
  * Keyboard Matrix Assignments
  *
@@ -152,4 +156,4 @@
 
 /* Bootmagic Lite key configuration */
 //#define BOOTMAGIC_LITE_ROW 0
-//#define BOOTMAGIC_LITE_COLUMN 0
\ No newline at end of file
+//#define BOOTMAGIC_LITE_COLUMN 0
diff --git a/keyboards/mechwild/puckbuddy/keymaps/default/keymap.c b/keyboards/mechwild/puckbuddy/keymaps/default/keymap.c
index bb5313fcb2..643fdb2818 100644
--- a/keyboards/mechwild/puckbuddy/keymaps/default/keymap.c
+++ b/keyboards/mechwild/puckbuddy/keymaps/default/keymap.c
@@ -34,10 +34,10 @@ enum layer_names {
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
     /* Base */
     [_BASE] = LAYOUT(
-    KC_MUTE,  MO(_FN3), MO(_FN2), MO(_FN1),       LGUI(KC_D),
-    KC_HOME,                                      KC_PGUP,
-    KC_END,                                       KC_PGDN,
-    KC_BTN3,  KC_BTN1, KC_BTN2, KC_BTN2, KC_BTN1, DPI_FINE
+    RGB_TOG,          KC_NO,   KC_NO,   DPI_DN,              DPI_UP,
+    LGUI(KC_DOWN),                                           LCA(KC_TAB),
+    KC_WBAK,                                                 KC_WH_U,
+    KC_WHOM,          KC_BTN1, KC_BTN2, KC_WH_L, KC_WH_R,    KC_WH_D
     ),
     [_FN1] = LAYOUT(
     RGB_TOG, KC_TRNS, KC_TRNS, KC_TRNS,          RESET,
@@ -57,4 +57,4 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
     KC_TRNS,                                     KC_TRNS,
     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
     )
-};
\ No newline at end of file
+};
diff --git a/keyboards/mechwild/puckbuddy/keymaps/via/keymap.c b/keyboards/mechwild/puckbuddy/keymaps/via/keymap.c
index bb5313fcb2..643fdb2818 100644
--- a/keyboards/mechwild/puckbuddy/keymaps/via/keymap.c
+++ b/keyboards/mechwild/puckbuddy/keymaps/via/keymap.c
@@ -34,10 +34,10 @@ enum layer_names {
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
     /* Base */
     [_BASE] = LAYOUT(
-    KC_MUTE,  MO(_FN3), MO(_FN2), MO(_FN1),       LGUI(KC_D),
-    KC_HOME,                                      KC_PGUP,
-    KC_END,                                       KC_PGDN,
-    KC_BTN3,  KC_BTN1, KC_BTN2, KC_BTN2, KC_BTN1, DPI_FINE
+    RGB_TOG,          KC_NO,   KC_NO,   DPI_DN,              DPI_UP,
+    LGUI(KC_DOWN),                                           LCA(KC_TAB),
+    KC_WBAK,                                                 KC_WH_U,
+    KC_WHOM,          KC_BTN1, KC_BTN2, KC_WH_L, KC_WH_R,    KC_WH_D
     ),
     [_FN1] = LAYOUT(
     RGB_TOG, KC_TRNS, KC_TRNS, KC_TRNS,          RESET,
@@ -57,4 +57,4 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
     KC_TRNS,                                     KC_TRNS,
     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
     )
-};
\ No newline at end of file
+};
diff --git a/keyboards/mechwild/puckbuddy/puckbuddy.c b/keyboards/mechwild/puckbuddy/puckbuddy.c
index 65e939166d..eeb5e20db3 100644
--- a/keyboards/mechwild/puckbuddy/puckbuddy.c
+++ b/keyboards/mechwild/puckbuddy/puckbuddy.c
@@ -6,12 +6,6 @@
 #ifndef GLIDEPOINT_DPI_OPTIONS
 #    define GLIDEPOINT_DPI_OPTIONS \
         { 400, 800, 1200, 1600, 2000, 2400, 2800, 3200, 3600, 4000 }
-#    ifndef GLIDEPOINT_DPI_DEFAULT
-#        define GLIDEPOINT_DPI_DEFAULT 1
-#    endif
-#endif
-#ifndef GLIDEPOINT_DPI_DEFAULT
-#    define GLIDEPOINT_DPI_DEFAULT 1
 #endif
 
 keyboard_config_t keyboard_config;
@@ -82,7 +76,7 @@ static void render_name(void) {     // Render Puckbuddy "Get Puck'd" text
     oled_set_cursor(0,3);
     oled_write_P(name_4, false);
 
-}    
+}
 
 static void render_logo(void) {     // Render MechWild "MW" Logo
     static const char PROGMEM logo_1[] = {0x97, 0x98, 0x99, 0x9A,0x00};
@@ -137,7 +131,7 @@ bool oled_task_kb(void) {
         }
         led_t led_state = host_keyboard_led_state();
         oled_set_cursor(8,0);
-        oled_write_P(led_state.scroll_lock ? PSTR("SCRLK") : PSTR("     "), false);
+        oled_write_P(led_state.scroll_lock ? PSTR("SCRLK") : PSTR("uf2  "), false);
         oled_set_cursor(8,1);
         oled_write_P(led_state.num_lock ? PSTR("NLCK ") : PSTR("     "), false);
         oled_write_P(led_state.caps_lock ? PSTR("CAPS ") : PSTR("     "), false);
@@ -207,4 +201,4 @@ void keyboard_post_init_kb(void) {
     keyboard_post_init_user();
     rgblight_toggle_noeeprom();     //double toggle post init removes the weirdness with rgb strips having a yellow first LED
     rgblight_toggle_noeeprom();
-}
\ No newline at end of file
+}
diff --git a/keyboards/mechwild/puckbuddy/rules.mk b/keyboards/mechwild/puckbuddy/rules.mk
index d9b8e45488..3d5a6caf57 100644
--- a/keyboards/mechwild/puckbuddy/rules.mk
+++ b/keyboards/mechwild/puckbuddy/rules.mk
@@ -2,7 +2,7 @@
 MCU = STM32F401
 
 # Bootloader selection
-BOOTLOADER = stm32-dfu
+BOOTLOADER = tinyuf2
 
 # Build Options
 #   change yes to no to disable
@@ -22,4 +22,4 @@ OLED_DRIVER = SSD1306       # OLED Driver
 DIP_SWITCH_ENABLE = yes     # Dip Switch Enabled
 
 POINTING_DEVICE_ENABLE = yes                    # Pointing Device Enabled
-POINTING_DEVICE_DRIVER = cirque_pinnacle_spi    # Pointing Device Driver
\ No newline at end of file
+POINTING_DEVICE_DRIVER = cirque_pinnacle_spi    # Pointing Device Driver
